.travis.yml

```diff
@@ -139,3 +139,5 @@ before_install:
   - opam install --yes ocp-indent
   # Crystal
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install crystal-lang; fi
+  # Bash
+  - pip install beautysh
```

Comment:
Great! This solved Travis CI build (for Linux). Mac is failing for unrelated reasons.